user/tests/wiring/no_fixture/pwm.cpp

```diff
@@ -29,15 +29,26 @@
 
 uint8_t pwm_pins[] = {
 
-#if defined(STM32F2XX)
+#if (PLATFORM_ID == 0) // Core
```

Comment:
lovely! I'm surprised we didn't hit the need to specify the PWM pins per platform before :-)